templates/test/dev/custom-builds/patches/kubeadm-controlplane-bootstrap.yaml

```diff
@@ -2,66 +2,72 @@
   path: /spec/kubeadmConfigSpec/files/-
   value:
     content: |
-        #!/bin/bash
+      #!/bin/bash
 
-        set -o nounset
-        set -o pipefail
-        set -o errexit
+      set -o nounset
+      set -o pipefail
+      set -o errexit
+      [[ $(id -u) != 0 ]] && SUDO="sudo" || SUDO=""
 
-        systemctl stop kubelet
-        declare -a BINARIES=("kubeadm" "kubectl" "kubelet")
-        az login --identity
-        for BINARY in "$${BINARIES[@]}"; do
-          echo "* installing package: $${BINARY} ${KUBE_GIT_VERSION}"
-          az storage blob download --blob-url "https://${AZURE_STORAGE_ACCOUNT}.blob.core.windows.net/${AZURE_BLOB_CONTAINER_NAME}/${KUBE_GIT_VERSION}/bin/linux/amd64/$${BINARY}" -f "/usr/bin/$${BINARY}" --auth-mode login
-        done
-        systemctl restart kubelet
+      # This test installs release packages or binaries that are a result of the CI and release builds.
+      # It runs '... --version' commands to verify that the binaries are correctly installed
+      # and finally uninstalls the packages.
+      # For the release packages it tests all versions in the support skew.
+      LINE_SEPARATOR="*************************************************"
+      echo "$${LINE_SEPARATOR}"
+      CI_VERSION=${CI_VERSION}
```

Comment:
@chrischdi @mboersma Why are the custom-builds templates now using `CI_VERSION` instead of custom-built artifacts? These templates are used in k/k presubmit jobs which test unmerged PRs, so using `CI_VERSION` won't exercise changes made by those PRs.

Reply:
I think they already did before?

CI_VERSION here is used for k8s binaries and images?!

Maybe I overlooked something here though and might be wrong.

The TL/DR of these changes were: the image re-tagging did not work anymore, leading to kubeadm fail.